web/src/main/java/com/graphhopper/srlarray/SpatialRuleContainer.java

```diff
@@ -26,7 +28,7 @@
  */
 class SpatialRuleContainer {
 
-    protected final Set<SpatialRule> rules = new LinkedHashSet<>();
+    final Set<SpatialRule> rules = new LinkedHashSet<>();
```

Comment:
Because the whole class is package-private, so protected on a method doesn't do anything. Also the class doesn't have sub-classes (and shouldn't), so protected doesn't do anything at all.